mac os build error

(Please complete the following information, and then delete this line)

**Affects versions :**
 - OS: Mac
 - Version of Xournal++: 1.11


**Describe the bug**

```
~/xournalpp/src/core/pdf/base/XojPdfPage.h:19:10: fatal error: 'cairo/cairo.h' file not found
```

**To Reproduce**

https://github.com/xournalpp/xournalpp/blob/master/readme/MacBuild.md

```
brew install cmake pkg-config gtk+3 poppler librsvg adwaita-icon-theme libzip portaudio libsndfile
mkdir build
cd build
cmake ..
make 
```

**Expected behavior**

```shell
~/xournalpp/src/core/control/AudioController.h:23:1: note: did you mean struct here?
class AudioRecorder;
^~~~~
struct
In file included from ~/xournalpp/src/core/control/Control.cpp:37:
In file included from ~/xournalpp/src/core/control/xojfile/LoadHandler.h:24:
In file included from ~/xournalpp/src/core/model/Document.h:27:
In file included from ~/xournalpp/src/core/pdf/base/XojPdfDocument.h:19:
In file included from ~/xournalpp/src/core/pdf/base/XojPdfDocumentInterface.h:19:
~/xournalpp/src/core/pdf/base/XojPdfPage.h:19:10: fatal error: 'cairo/cairo.h' file not found
#include <cairo/cairo.h>  // for cairo_region_t, cairo_t
         ^~~~~~~~~~~~~~~
3 warnings and 1 error generated.
make[2]: *** [src/core/CMakeFiles/xournalpp-core.dir/control/Control.cpp.o] Error 1
make[1]: *** [src/core/CMakeFiles/xournalpp-core.dir/all] Error 2
make: *** [all] Error 2
```

> Does it build if you replace `cairo/cairo.h` by `cairo.h` in `XojPdfPage.h` and `RenderJob.h`?

yes , thanks !
replace `XojPdfPage.h` to `#include <cairo.h>`  can work  success

```
xournalpp/src/core/pdf/base/XojPdfPage.h:19:10: fatal error: 'cairo/cairo.h' file not found
xournalpp/src/core/control/jobs/RenderJob.h:14:10: fatal error: 'cairo/cairo.h' file not found
```

--- a/src/core/control/jobs/RenderJob.h
+++ b/src/core/control/jobs/RenderJob.h
@@ -11,7 +11,7 @@
 
 #pragma once
 
-#include <cairo/cairo.h>  // for cairo_surface_t
+#include <cairo.h>    // for cairo_surface_t
 #include <gtk/gtk.h>  // for GtkWidget
 
 #include "Job.h"  // for Job, JobType

--- a/src/core/pdf/base/XojPdfPage.h
+++ b/src/core/pdf/base/XojPdfPage.h
@@ -16,7 +16,7 @@
 #include <string>     // for string
 #include <vector>     // for vector
 
-#include <cairo/cairo.h>  // for cairo_region_t, cairo_t
+#include <cairo.h>  // for cairo_region_t, cairo_t
 
 /// Determines how text is selected on a user action.
 enum class XojPdfPageSelectionStyle : uint8_t {

